internal/ingest/linksystem.go

```diff
@@ -277,25 +174,34 @@ func (ing *Ingester) storageHook(pubID peer.ID, c cid.Cid) {
 	// Remove the content block from the data store now that processing it
 	// has finished.  This prevents storing redundant information in
 	// several datastores.
-	err = ing.ds.Delete(context.Background(), dsKey(c.String()))
+	err = ing.ds.Delete(context.Background(), dsKey(entryChunkCid.String()))
```

Comment:
nit: could use key from above

Reply:
Should this be deleted in a `defer` function immediately after it is read (at top of function), so that it gets cleaned up in the case of error, or do we want it to remain in datastore if this function exits with error before getting here?

Reply:
done in 32df0a3